src/common/data/models/interfaces/RecipientDescribing.ts

```diff
@@ -62,15 +62,15 @@ export function makeContactRecipientDescription(
   return {
     id: data.id,
     kind: RecipientKind.CONTACT,
-    displayedName: data.contactName,
-    walletName: data.contactsWalletName,
-    avatarImageSource: null,
+    displayedName: data.contactName || data.displayedName,
```

Comment:
Why do we need multiple variable names for `displayName` ?

Can we not have jus` data.displayName?`

Reply:
Actually, this object is referring two times when the user picks data from System contact then the key name is "contactName" and when on SendToContact flow again this object is referring then we have displayName